src/gtc/passes/oir_optimizations/caches.py

```diff
@@ -186,6 +186,20 @@ def requires_fill(field: str) -> bool:
             return {field for field in filling_fields if not requires_fill(field)}
 
         pruneable = set.intersection(*(pruneable_fields(section) for section in node.sections))
+
+        # Consider accesses outside of loop interval
+        # Note: those accesses would not require a fill in the whole loop interval in in theory
```

Comment:
```suggestion
        # Note: those accesses would not require a fill in the whole loop interval in theory
```

Reply:
I know it's annoying, but please expand the comment with details, maybe with example, such that a non-GridTools developers can understand.

Reply:
Done. Hope that’s understandable enough.

Reply:
time will tell ;)